index.php

```diff
@@ -15,11 +15,25 @@
 <body>
     <main>
         <h1>BEHOLD!! Some deities and stuff:</h1>
-        <div>
+        <div class="displayDeities">
```

Comment:
rename this class to something more nouny such as displayDeitiesSection or something